deploy/iso/minikube-iso/Dockerfile

```diff
@@ -30,14 +33,12 @@ RUN apt-get update && apt-get install -y \
 		locales \
 		rsync \
 		dumb-init \
-		&& rm -rf /var/lib/apt/lists/*
+		golang-go \
```

Comment:
Which version of go is this? I think we still want to install that separately to control it outside of the Ubuntu release.

Reply:
Ah, sgtm.